app/Http/Controllers/Auth/AuthController.php

```diff
@@ -33,9 +40,17 @@ public function login()
 
     public function loginPost()
     {
-        \Auth::attempt(request()->only(['email', 'password']), request('remember'));
+        if (Auth::attempt(['email' => request('email'), 'password' => request('password'), 'status' => 'active'], request('remember'))) {
+            Notifications::success('Successfully login', 'top');
+
+            return redirect()->route('index');
+        } else {
+            Notifications::error('Access error. The reasons may be incorrect email password pair or unconfirmed registration<br>
```

Comment:
I know. You changed this method. And your changes will I see in the conflicts. Your changes are saved